edit.c

```diff
@@ -606,17 +615,23 @@ void edit_char(uint32_t ch, _Bool control, uint8_t flags)
             break;
         }
 
+#ifdef KEY_LEFT_TAB
```

Comment:
I still disagree with this ifdef. For two reasons:
 - it's confusing (people looking at it in the future won't get the difference between KEY_TAB and KEY_LEFT_TAB);
 - it leaks platform-defined code into cross-platform code. µTox seems currently structured so that all platform-defined code is in one of the win32, xlib or android directories, but this ifdef basically adds xlib-only code to the part which is supposed to be crossplatform.

What I'd do is(and I've tested this and it works):
 - Remove this ifdef and its content
 - Remove the definition of KEY_LEFT_TAB in xlib/main.h
 - In xlib/event.c, around line 278, add this:
```c
if(sym == XK_ISO_Left_Tab){
    sym = XK_Tab;
}
```
That normalizes what I see as just an idiotic quirk of xlib(2 tab keycodes) without complicating the crossplatform code.

For the rest, I've tested this on Xubuntu and wine and everything works, and it's really nice to have this feature. Great work!